### PR TITLE
NetApp update swagger for R4.5

### PR DESCRIPTION
**Pull request by tomasm9-zz**
### Summary of Changes
- Added async responses to: 
-- Account update
-- Pools update
-- Volumes update
-- Snapshots update
-- Snapshots delete

- Removed default value from size in poolProperties
- Protocol types changed from an array inside a string, to an array of strings
- Snapshot id allows resource id in volumeProperties
-- Added to the regex

-Updated the API version

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by tomasm9** [comment]
For some reason one of the pipeline checks is failing(Breaking Changes). It's complaining about a line that is identical to last versions line.  All other checks pass, and json validation doesn't complain.

Are you familiar with the error we're getting?

**Review by amarzavery — APPROVED** [review]

**Comment by amarzavery** [comment]
@tomasm9 - Can you summarize the changes made in the new api version?

**Review by amarzavery — CHANGES_REQUESTED** [review]
Please remove the test-swagger folder which contains the C# code. This repository only contains the swagger spec. Please send a PR to https://github.com/azure/azure-sdk-for-net repo for the C# sdk that is generated from this swagger spec.

**Comment by ravbhatnagar** [comment]
looks like the parameters are being included for each API. earlier they were at the level of the path and now they are being added for each http verb. Signing off from ARM side.

**Comment by tomasm9** [comment]
I added a quick summary in the description with the changes

@amarzavery the C# code was accidentally added in, should be removed now :) 

> looks like the parameters are being included for each API. earlier they were at the level of the path and now they are being added for each http verb. Signing off from ARM side.

@ravbhatnagar should be reverted now

**Review by amarzavery — APPROVED** [review]
